### DISCUSSION TITLE
MiKTeX 21.6.28

### DISCUSSION THREAD

**Discussion by edocevoli** [MiKTeX/miktex#857] [General]
#### Common TEXMF root directories in a private (current user) installation

In the past, common TEXMF root directories (e.g., `C:\Program Files\MiKTeX`)
were in the input search path, even if MiKTeX was installed for the current user only.
As of 21.6.28, common TEXMF root directories are excluded from the input search path.

This should have no impact for most users, because these directories usually
do not exist when MiKTeX is installed for the current user.

### Fixed issues

- [841](https://github.com/MiKTeX/miktex/issues/841): When a package have been uninstalled, its directories are not removed
- [844](https://github.com/MiKTeX/miktex/issues/844): Documentation not found
- [846](https://github.com/MiKTeX/miktex/issues/846): miktex update causes issues “go to source/pdf” in TeXstudio
  

<hr /><em>This discussion was created from the release <a href='https://github.com/MiKTeX/miktex/releases/tag/21.6.28'>MiKTeX 21.6.28</a>.</em>

**Comment by pabloweigandt** [comment]
I want to use Lyx with an A&A template. The massage is saying that I have to install aa.cls file. I try to do that in MiKTex 21.6 but I fail !!! Please, help me.

**Comment by pabloweigandt** [comment]
Any advice?
